### ISSUE TITLE
Breadcrumb navigation

### ISSUE THREAD

**Issue by thatbudakguy** [pod4lib/aggregator#478]
there's currently only one type of page in the app that uses breadcrumb navigation; the [upload show page](https://pod.stanford.edu/organizations/duke/uploads/31539):
![Screen Shot 2022-03-16 at 14 42 45](https://user-images.githubusercontent.com/4924494/158696057-48b21b6b-efeb-4b9e-ac2e-2a54707cac26.png)

the navigation isn't formatted well and should probably extend horizontally instead of wrapping. but, more generally: do we want to make use of the breadcrumb navigation elsewhere? it seems inconsistent to limit it to one page.

we currently have a few types of pages that are somewhat nested (organization token management, for example) that could benefit from the breadcrumbs. on the other hand, many of those pages have a nice large header that shows which organization you're under when you're visiting the page, which might help accomplish the same thing:

![Screen Shot 2022-03-16 at 14 45 45](https://user-images.githubusercontent.com/4924494/158696437-55eb4375-0a2f-40b2-818c-bcc9be0e4fea.png)

but, this header doesn't appear on the upload show page, even though uploads are tied to an organization.

one possibility: ditch the breadcrumb nav for now, and make sure we use the organization header with the logo for all pages underneath an organization, so that the user knows where they are.

**Comment by ggeisler** [comment]
@thatbudakguy Yep, I also noticed the breadcrumb (for the first time) when following the links on the Activity page. This is a good ticket to remind us of the issue. I probably will wait to offer a breadcrumb-specific opinion until I finish -- and the team discusses -- ideas around reorganizing all of the application pages, because one issue I'm trying to deal with there is the navigation around the Organization pages. So that proposed solution might eliminate the need for breadcrumbs, or might lead us to adding breadcrumbs throughout the app. TBD.

**Comment by ggeisler** [comment]
We've done a pretty thorough reorg/redesign of the application pages (and fixed things like Upload show page example in the original issue description) and the potential need for breadcrumbs seems reduced so I'm going to close this. 

I don't necessarily think breadcrumbs couldn't be helpful but they don't seem **necessary** at this point, so we can probably just give the application time to get a decent amount of use and revisit breadcrumbs if we get feedback indicating the application is hard to navigate or understand.
